CMakeLists.txt

```diff
@@ -148,7 +149,7 @@ if (${CODYCO_USES_OROCOS_BFL_BERDY})
 endif()
 
 #mex wholeBodyModel interface
-if (${CODYCO_USES_KDL} AND ${CODYCO_USES_MATLAB})
+if (${CODYCO_USES_KDL} AND ${CODYCO_USES_MATLAB} AND ${CODYCO_USES_MEX_WHOLEBODYMODEL})
```

Comment:
`${CODYCO_USES_KDL}` is not needed since https://github.com/robotology/yarp-wholebodyinterface/pull/63 , so we can remove it. I think it is sufficient to check if `CODYCO_USES_MEX_WHOLEBODYMODEL` is set, because if the user set it to explicitly to ON it is not intuitive that also `CODYCO_USES_MATLAB` needs to be set to ON.